### ISSUE TITLE
Syntax error in file

### ISSUE THREAD

**Issue by olalonde** [siddharthkp/auto-install#76]
```
$ auto-install
Could not parse src/bin/server.js. There is a syntax error in file
```

`auto-install` is reporting a syntax error for the following file:

Source of `src/bin/server.js`:

``` javascript
/* eslint-disable no-console */
import 'babel-polyfill'
import kue from 'kue'
import initServer from '../'
import basicAuthMiddleware from '../middlewares/basic-auth'

/* eslint-disable global-require */
if (process.env.LONGJOHN) {
  require('longjohn')
}

const server = initServer()
const { port, basicAuth } = server.config
const app = server.app

app.use('/kue', basicAuthMiddleware(basicAuth.username, basicAuth.password))
app.use('/kue', kue.app)

server.start().then(() => {
  console.info(`==> 🌎  Listening on port ${port}. Open up http://localhost:${port}/ in your browser.`)
})
```

**Comment by siddharthkp** [comment]
@olalonde You have landed on a bug. I'll fix it tonight 👍 

Thanks for reporting

**Comment by siddharthkp** [comment]
@olalonde Doesn't work with both `import` and `require` in one file 😔 

Has to do something with using `detective` and `es6detective` together in [`getModulesFromFile`](https://github.com/siddharthkp/auto-install/blob/master/src/helpers.js#L66-L80)

For now, you can change `require('longjohn')` to use `import`. Meanwhile, I'll find the bug and fix it.

**Comment by siddharthkp** [comment]
@olalonde Fixed here https://github.com/siddharthkp/auto-install/pull/77 😄 

Published `1.7.2`

**Comment by olalonde** [comment]
Thanks!
